api/metrics/views.py

```diff
@@ -154,3 +160,42 @@ def metric_type(self):
     @property
     def metric(self):
         return PreprintDownload
+
+class RawMetricsView(GenericAPIView):
+
+    permission_classes = (
+        drf_permissions.IsAuthenticated,
+        IsRawMetricsUser,
+        TokenHasScope,
+    )
+
+    required_read_scopes = [CoreScopes.METRICS_BASIC]
+    required_write_scopes = [CoreScopes.METRICS_RESTRICTED]
+
+    view_category = 'raw-metrics'
+    view_name = 'raw-metrics-view'
+
+    serializer_class = RawMetricsSerializer
+
+    @require_switch(ENABLE_RAW_METRICS)
+    def delete(self, request, *args, **kwargs):
+        raise ValidationError('DELETE not supported. Use GET/POST/PUT')
+
+    @require_switch(ENABLE_RAW_METRICS)
+    def get(self, request, *args, **kwargs):
+        connection = get_connection()
+        return JsonResponse(connection.search())
```

Comment:
This is nice and simple. Does it pass along query parameters as well?

Reply:
Ah. It does not. Thank you